2-data.sql

```diff
@@ -0,0 +1,9 @@
+set dateformat dmy
```

Comment:
Can't run file multiple times.

Reply:
I added a delete statement, should work now
Thank you